### PR TITLE
chore: merge develop into offline extraction feature branch

### PR DESCRIPTION
**Pull request by admbtlr**

### CODE REVIEW

File: aether-kernel/setup.cfg [hunk 1]
```diff
@@ -14,4 +14,4 @@ omit = *migrations*, *tests*
 [coverage:report]
 omit = *migrations*, *tests*, *settings.py
 show_missing = true
-fail_under = 100
+fail_under = 95
```
Comment (obdulia-losantos) [inline aether-kernel/setup.cfg:17]: One step back...

Comment (admbtlr) [reply aether-kernel/setup.cfg:17]: Ha! True. So what do you think? Should we accept 95%, or write the extra tests?

Comment (obdulia-losantos) [reply aether-kernel/setup.cfg:17]: We can accept 95% but I would suggest to write the missing tests in a near future.

Comment (admbtlr) [reply aether-kernel/setup.cfg:17]: Agreed. Can you approve then please?

**Review by obdulia-losantos — APPROVED** [review]

**Pull request merged**